client/actions/documentActions.js

```diff
@@ -0,0 +1,94 @@
+import axios from 'axios';
+import * as actionTypes from './actionTypes';
+import { beginAjaxCall } from './ajaxStatusActions';
+import handleError from '../utils/errorHandler';
+
+export function getDocumentSuccess(documents, pageData, offset) {
+  return {
+    type: actionTypes.GET_DOCUMENTS_SUCCESS,
+    documents,
+    pageData,
+    offset
+  };
+}
+
+export function getDocuments(offset = 0) {
+  return (dispatch) => {
+    dispatch(beginAjaxCall());
+    return axios.get(`/documents?offset=${offset}`)
+      .then((res) => {
+        dispatch(getDocumentSuccess(res.data.documents,
+          res.data.pageData, offset));
```

Comment:
If all the arguments do not fit in one line (80characters) then you need a new line per argument.